src/components/ContactFormNewsletter.js

```diff
@@ -26,48 +18,83 @@ const NewsletterForm = () => {
     }
 
     useEffect(() => {
-        if (!widget.current && container.current) {
-            widget.current = new WidgetInstance(container.current, {
-                startMode: "auto",
-                doneCallback: doneCallback,
-                errorCallback: errorCallback
-            });
-        }
-
-        return () => {
-            if (widget.current !== undefined) widget.current.destroy();
-        }
-    }, [container]);
+        setInputs(values => ({ ...values, [turnstileCaptchaFieldName]: token }))
+    }, [token]);
 
     const handleSubmit = (evt) => {
-        evt.preventDefault();
-        const uri = new URI();
-        uri.addQuery("form-name", evt.target.getAttribute("name"));
-        uri.addQuery(inputs);
+        try
```

Comment:
@matiasperrone-tipit this code is repeated on all the 3 forms
i do think for the sake on manageability and to respect DRY principle we need to extract this common logic on a custom hook
like **useTurnstileFormSubmit**

and use it form like this

```
const { handleSubmit, submitting } = useTurnstileFormSubmit();
<form onSubmit={handleSubmit(inputs, { turnstileFieldName, onSuccess: () => setSuccess(true) })} />
```